After update to 12.X problem

I use angular 14.1.3 version. 

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @angular/common@^13.0.0 | ^14.0.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

```

--- a/projects/ngx-captcha-lib/package.json
+++ b/projects/ngx-captcha-lib/package.json
@@ -27,7 +27,7 @@
     "prevent-publish": "node prevent-publish"
   },
   "peerDependencies": {
-    "@angular/common": "^13.0.0 | ^14.0.0",
-    "@angular/core": "^13.0.0 | ^14.0.0"
+    "@angular/common": "^13.0.0 || ^14.0.0",
+    "@angular/core": "^13.0.0 || ^14.0.0"
   }
 }

